Readme.md

```diff
@@ -36,6 +38,27 @@ Alternately you could start INSTEAD with preloaded game, for example:
 $ sdl-instead path/to/old-mixer-tales
 ```
 
+### JavaScript INSTEAD Client
+
+There's an [instead-js] project we also support. To run it, follow the
+instructions in the instead-js repository. Here's a quick excerpt:
```

Comment:
Couldn't find an authoritative source for that, but in my understanding, excerpts can't be "quick". I found this [list of alternatives](http://adjective1.com/for-excerpt/).

Age-old Google test corroborates:
* ["quick excerpt"](https://encrypted.google.com/search?hl=en&q="quick%20excerpt") — 33,300 results;
* ["brief excerpt"](https://encrypted.google.com/search?hl=en&q="brief%20excerpt") — 337,000 results;
* ["small excerpt"](https://encrypted.google.com/search?hl=en&q="small+excerpt") — 410,000 results.

Finally, what you provide is more of a gist than an excerpt.